ui/src/CollectionExerciseDetails.js

```diff
@@ -568,6 +568,20 @@ class CollectionExerciseDetails extends Component {
             </div>
           </DialogContent>
         </Dialog>
+        {(this.state.authorisedActivities.includes("LOAD_BULK_REFUSAL") ||
+          this.state.authorisedActivities.includes(
+            "VIEW_BULK_REFUSAL_PROGRESS"
```

Comment:
this code could get super unwieldy as we add more and more bulk processors, so I would suggest writing it like this...

```js
["LOAD_BULK_REFUSAL", "VIEW_BULK_REFUSAL_PROGRESS"].some( p => this.state.authorisedActivities.includes(p) )
```